### DISCUSSION TITLE
Serene MVC absolute version VS 2022 installation problem

### DISCUSSION THREAD

**Discussion by Saad2025** [serenity-is/Serenity#6612] [Help]
I want to use Serene MVC absolute version on VS 2022 but I get installation problem every time I try to install it in vs 2022 please could any body help me with this problem

**Comment by Saad2025** [comment]
I need to use absolute version for importance

**Comment by volkanceylan** [comment]
it's not compatible with vs2022, install an older version of VS, like 2019, 2017 etc.

**Reply by Saad2025** [reply]
why it's not compatible with vs 2022 ? I need to know the reason please if you don't mind

**Reply by VictorTomaili** [reply]
You need to ask it to Microsoft instead. It is not related to us.
